INSTALL.md

```diff
@@ -31,3 +31,13 @@ To install from head with [Go] v1.10.1 or higher:
 ```
 go get github.com/kubernetes-sigs/kustomize
 ```
+
+## Installation using the kustomize.sh wrapper
+
+You can use the [kustomize.sh](scripts/kustomize.sh) script to
```

Comment:
Can you add a command for using this script. Maybe similar to
`curl https://api.github.com/repos/kubernetes-sis/kustomize/scripts/kustomize.sh`

Reply:
Maybe there is a misunderstanding about the intention of this script: it is a wrapper script that is used in place of the kustomize binary, and not just an installer.

The idea is that you can commit this to your project git repo (say in myproject/tools/kustomize.sh), and then you can call it everywhere from there, for example also in ci/cd build pipelines.

Showing a curl command doesn't really help, because it's not something that you only execute once. You should just download the script, possibly modify it to your liking, and put it in your git repo.